features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,22 @@
+Given("I am on the {string} Page") do |page_name|
+  visit page_path(page_name)
+end
+
+Then("I fill in {string} with {string}") do |element, content|
+  fill_in(element, :with => content)
```

Comment:
Please use new hash syntax

Reply:
Got it! 
We forgot and looked in the capybara documentation.